adapters/yeahmobi/yeahmobi.go

```diff
@@ -187,3 +203,10 @@ func getBidType(impId string, imps []openrtb2.Imp) openrtb_ext.BidType {
 	}
 	return bidType
 }
+
```

Comment:
As you mentioned that zmaticoo is a new adapter. Could you please help us in understanding why is code being changed for yeahmobi?

Reply:
zMaticoo is a new subject with its own independent entity. Yeahmobi is another subject and brand. The code of Yeamobi Adapter was written by me before, but now this part of the code has compatibility issues, so I made modifications.If this operation is not allowed, I can roll back this part of the code and let the current Yeahmobi Adapter developer make modifications。

Reply:
yes please. Requesting you to please remove these changes and raise another PR if you are the owner of yeahmobi adapter.

Reply:
ok，i have  removed these codes，please review these codes。
thank you ~